proto/kvrpcpb.proto

```diff
@@ -938,6 +942,12 @@ enum APIVersion {
     V2 = 2;
 }
 
+message SourceStmt {
+    uint64 start_ts = 1;
+    uint64 connection_id = 2;
+    uint64 stmt_id = 3;
+}
```

Comment:
Since the `connection_id` can be identical across different TiDB servers, it may be necessary to use a unique identifier such as `uuid` to indicate which statement and instance this query originates from.
Besides, how should the `stmt_id` be generated?

Reply:
> how should the stmt_id be generated

Each connection generates its own stmt_id. 
Merging them together into a UUID can make it harder for humans to read. Since it's only used in tests, the performance gain doesn't matter a lot.